src/Rules/IRuleInfo.cs

```diff
@@ -170,5 +176,15 @@ public class RuleInfo : IRuleInfo
         public IReadOnlyList<string> EducationPrinciples { get; }
 
         public string HtmlNote { get; set; }
+
+        public IRuleInfo WithOverridenSeverity(RuleIssueSeverity newSeverity)
```

Comment:
Maybe combine the 2 override methods? They're not super useful on their own and create an extra unnecessary object that's easy to avoid

Reply:
they're not used separately is what I rather meant